backend/src/controllers/project_status.js

```diff
@@ -1,6 +1,17 @@
 const useController = require("./useController/index.js");
 const model = require("../models/project_status");
-const what = { single: "project_status", plural: "project_statuses" };
+const what = { single: "project", plural: "projects" };
```

Comment:
I feel like i might have given you bad advice here, but because how we are using, this, it might not be a problem functionally, but an issue in readability.